### PR TITLE
Ocrvs 2376: Left navigation added on mobile view

### PR DESCRIPTION
**Pull request by emon-swe-sust**
Screenshot from 2022-01-18 17-47-15

Left navigation added on mobile view, some other files are changed.

### CODE REVIEW

File: packages/client/src/components/interface/Navigation.tsx [hunk 1]
```diff
@@ -0,0 +1,648 @@
...
+  performance: 'Performance',
+  team: 'Team',
+  configuration: 'Configuration',
+  certificatesConfiguration: 'Certificates',
+  applicationSettings: 'Application Settings',
+  settings: 'Settings',
+  logout: 'Logout'
+}
+
+const goToPerformanceView = (props: IFullProps) => {
```
Comment (rikukissa) [inline packages/client/src/components/interface/Navigation.tsx:197]: What do you think about having these two functions in `packages/client/src/navigation/index.ts` and having the function signatures just as

```
const goToPerformanceView = (userDetails) => {
const goToTeamView = (userDetails) => {
``` 

as there they could directly refer to the action creators you're using internally here?

File: packages/client/src/i18n/messages/constants.ts [hunk 2]
```diff
@@ -185,6 +186,11 @@ const messagesToDefine: IConstantsMessages = {
     description: 'Informant Label',
     id: 'constants.informant'
   },
+  applicationName: {
+    defaultMessage: 'Open CRVS',
```
Comment (rikukissa) [inline packages/client/src/i18n/messages/constants.ts:190]:
```suggestion
    defaultMessage: 'OpenCRVS',
```

File: packages/client/src/views/OfficeHome/RegistrationHome.test.tsx [hunk 3]
```diff
@@ -136,11 +136,11 @@ beforeAll(async () => {
   await store.dispatch(checkAuth({ '?token': registerScopeToken }))
 })
 
-describe('RegistrationHome related tests', () => {
+describe('OfficeHome related tests', () => {
   it('sets loading state while waiting for data', async () => {
     const testComponent = await createTestComponent(
       // @ts-ignore
```
Comment (rikukissa) [inline packages/client/src/views/OfficeHome/RegistrationHome.test.tsx:142]: Could we get rid of the @ts-ignore's here while we're at it?

File: packages/components/src/components/icons/Configuration.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+ * graphic logo are (registered/a) trademark(s) of Plan International.
+ */
+import React from 'react'
+import { Meta, Story } from '@storybook/react'
+import { Configuration } from './Configuration'
+
+export default {
+  title: 'Components/icons/Configuration',
+  component: Configuration
+} as Meta
```
Comment (rikukissa) [inline packages/components/src/components/icons/Configuration.stories.tsx:19]: Do we need to explicitly cast these as `Meta`? What happens if you drop the `as Meta` cast?

Comment (emon-swe-sust) [reply packages/components/src/components/icons/Configuration.stories.tsx:19]: I was following the practice that @soumita-dsi followed on her previous stories, but I found that if I drop 'as Meta' , it works fine, but it is recommended in storybook documentation to cast these "as Meta" .

Comment (rikukissa) [reply packages/components/src/components/icons/Configuration.stories.tsx:19]: Odd that they're encouraging such a way of typing the objects. Could you link the documentation where it's stated here just to have it somewhere documented if someone asks later on? In my opinion, it's such a small thing anyways that I'm happy either way. I was mainly just wondering :)

Comment (emon-swe-sust) [reply packages/components/src/components/icons/Configuration.stories.tsx:19]: Here @rikukissa https://storybook.js.org/docs/react/api/csf#named-story-exports , you can see the examples.
But if you want I'll delete Casting as Meta anyway.

File: packages/client/src/components/interface/Navigation.tsx [hunk 5]
```diff
@@ -0,0 +1,605 @@
...
+  applicationGroup: 'applicationGroup',
+  menuGroup: 'menuGroup'
+}
+
+interface IUSER_SCOPE {
+  [key: string]: string[]
+}
+
+const USER_SCOPE: IUSER_SCOPE = {
+  FIELD_AGENT: [
```
Comment (rikukissa) [inline packages/client/src/components/interface/Navigation.tsx:87]: Notice that you're missing `DISTRICT_REGISTRAR` from here. Are there any other roles we should have here?

**Review by rikukissa — APPROVED** [review]

**Pull request merged**